Text/Pandoc/Definition.hs

```diff
@@ -209,31 +211,96 @@ instance Eq Format where
 instance Ord Format where
   compare (Format x) (Format y) = compare (T.toCaseFold x) (T.toCaseFold y)
 
+-- | The number of columns taken up by the row head of each row of a
+-- 'TableBody'. The row body takes up the remaining columns.
+newtype RowHeadColumns = RowHeadColumns Int
+  deriving (Eq, Ord, Show, Read, Typeable, Data, Generic, Num, Enum)
```

Comment:
just stumbled over this tweet... https://twitter.com/mxlambda/status/1248228970694402048?s=12 :-)